tests/test_matrix_utils.py

```
@@ -2262,6 +2375,464 @@ async def test_send_room_image():
+@patch("builtins.open")
+@patch("mmrelay.matrix_utils.json.load")
+@patch("mmrelay.matrix_utils._create_ssl_context")
+@patch("mmrelay.matrix_utils.matrix_client", None)
+@patch("mmrelay.matrix_utils.AsyncClient")
+@patch("mmrelay.matrix_utils.logger")
+async def test_connect_matrix_restore_login_uses_none_device_id(
+    _mock_logger,
+    mock_async_client,
+    mock_ssl_context,
+    mock_json_load,
+    mock_open,
```

Ruff (0.14.5)

Unused function argument: `mock_open`

(ARG001)